ui-v2/package.json

```diff
@@ -103,10 +104,12 @@
     "jsonlint": "^1.6.3",
     "lint-staged": "^9.2.5",
     "loader.js": "^4.7.0",
+    "mnemonist": "^0.30.0",
```

Comment:
Was this package required to use `tape`?

Reply:
Oh, good catch, weird I thought that came from the rebase of the discovery chain things, lemme take another look - I'm not even sure if I had to rebase this one or not.

Reply:
Not sure how that got in there, I'm using it in another branch, so it might have been from undo-ing, I've taken that out. Just going to wait and see if CI passes.